packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -73,18 +73,67 @@
 
 
 .block-editor-global-styles-background-panel__inspector-media-replace-container {
-	border: 1px solid $gray-300;
-	border-radius: 2px;
 	// Full width. ToolsPanel lays out children in a grid.
 	grid-column: 1 / -1;
 
+	& > div {
+		// Border styles.
+		border-left: 1px solid $gray-300;
+		border-right: 1px solid $gray-300;
+		border-bottom: 1px solid $gray-300;
+
+		&:not(:only-child):nth-child(1 of &) {
+			border-top-left-radius: $radius-block-ui;
+			border-top-right-radius: $radius-block-ui;
+			border-top: 1px solid $gray-300;
+			border-bottom-left-radius: 0;
+			border-bottom-right-radius: 0;
+		}
+
+		// Identify the last visible instance as placeholder items will not have this class.
+		&:not(:only-child):nth-last-child(1 of &) {
+			border-bottom-left-radius: $radius-block-ui;
+			border-bottom-right-radius: $radius-block-ui;
+			border-top: 0;
+			border-top-left-radius: 0;
+			border-top-right-radius: 0;
+		}
+
+		&:only-child {
+			border-top: 1px solid $gray-300;
+			border-radius: $radius-block-ui;
+		}
+
+		> div,
+		> div > div,
+		> div > button,
+		> div > div > button {
+			border-radius: inherit;
+		}
```

Comment:
This seems like a code smell to me. It looks like we're not using [`ItemGroup`](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-itemgroup--with-border) correctly. `<ItemGroup isBordered isSeparated>` is supposed to handle these border concerns. It should be wrapping both the "Image" and "Color" items, but right now it is only wrapping the "Image" item.

Reply:
Yeah, we should reconcile this

Reply:
Hi @mirka @mtias,

I have made the changes to wrap both the "Image" and "Color" items with ItemGroup. However, after resolving the recent merge conflicts, it has become confusing. If you can help here, I would appreciate it.